protocol-definitions/Client.yaml

```diff
@@ -1092,3 +1092,28 @@ methods:
           doc: |
             Authentication token bytes for the TPC channels
     response: {}
+  - id: 23
+    name: addCPGroupViewListener
+    since: 2.7
+    doc: |
+      Adds a CP Group view listener to a connection.
+    request:
+      retryable: false
+      partitionIdentifier: -1
+    response: {}
+    events:
+      - name: GroupsView
+        since: 2.8
+        params:
+          - name: version
+            type: long
+            nullable: false
+            since: 2.8
+            doc: |
+              The version number for this group view
+          - name: groupToMembers
+            type: Map_RaftGroupId_List_UUID # GroupID -> [Leader UUID, Follower UUID, Follower UUID, ...]
```

Comment:
This event needs to carry information for multiple groups, no one - I did consider having 2 maps, 1 for `GroupId` --> `Leader`, and another for `GroupId` -->`Followers`, but this seemed inefficient use of network as we'd be sending duplicate information (group IDs).

What makes you pessimistic about this approach? It's fairly standard to be able to rely on information order (1st element is X, remaining elements are Y), and we do it elsewhere in Hazelcast already.

Reply:
I can see that it looks like convenient but brings hidden problems. It is hard to debug, error prone and against to idea of using protocol. Depending on ordering must be done in binary level which is fine since implementation is fixed. However,  on higher level we can introduce new types without much that concern. This is the reason why protocol exist. 

My suggestion is introducing a new type like `RaftGroupList` which contains leader id and its followers as list. So it is roughly like , group id -> RaftGroupList {leaderid, followers} This is much safer.

Reply:
Added `RaftGroupInfo` in style similar to your suggestion, thanks 👍